models/business-unit/src/associate/generator.ts

```diff
@@ -0,0 +1,15 @@
+import { Reference } from '@commercetools-test-data/commons';
+import { fake, Generator } from '@commercetools-test-data/core';
+import * as AssociateRoleAssignment from '../associate-role-assignment';
+import { TAssociateDefault } from './types';
+
+// https://docs.commercetools.com/api/projects/business-units#associate
+
+const generator = Generator<TAssociateDefault>({
+  fields: {
+    associateRoleAssignments: [fake(() => AssociateRoleAssignment.random())],
+    customer: fake(() => Reference.random().typeId('customer')), // should use ReferenceDraft
```

Comment:
```suggestion
    customer: fake(() => ReferenceDraft.presets.customerReference()),
```

Reply:
I'm probably wrong, but I was under the impression that we do want `typename` in non-draft models?

Reply:
...which is to admit that my comment was in the wrong spot.

Reply:
This should be `Reference` not `ReferenceDraft` I was mostly going for the preset implementation

Reply:
Yes. Drafts use drafts. All the way down.